spec/spec_helper.rb

```diff
@@ -22,21 +24,27 @@ def establish_database_connection
   ActiveRecord::Base.establish_connection(
     adapter: "postgresql",
     host: ENV.fetch("PGHOST", "localhost"),
-    user: ENV.fetch("PGUSER", "postgres"),
-    password: ENV.fetch("PGPASSWORD", ""),
+    user: ENV.fetch("PGUSER", "ubuntu"),
```

Comment:
Does the user need to change here? Or was it always wrong?

Reply:
the readme file suggestion is ubuntu, so updated it to be same as that for testing